app/src/main/res/layout/listitem_user_equipment_set.xml

```diff
@@ -0,0 +1,263 @@
+                    app:srcCompat="@drawable/ic_element_ice" />
+
+                <TextView
+                    style="@style/TextSubHead"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/margin_large"
+                    android:text="@string/armor_detail_vs_ice" />
+
+                <TextView
+                    android:id="@+id/ice_value"
+                    style="@style/TextSubHead"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="@dimen/margin_large"
+                    android:gravity="end"
+                    tools:text="2" />
+            </LinearLayout>
+            <com.gatheringhallstudios.mhworlddatabase.components.DashedDividerView
+                android:layout_width="wrap_content"
+                android:layout_height="match_parent" />
+            <LinearLayout style="@style/EquipmentRow">
+
+                <ImageView
+                    style="@style/IconXSmall"
+                    android:layout_marginStart="@dimen/margin_large"
+                    app:srcCompat="@drawable/ic_element_dragon" />
+
+                <TextView
+                    style="@style/TextSubHead"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="@dimen/margin_large"
+                    android:text="@string/armor_detail_vs_dragon" />
+
+                <TextView
+                    android:id="@+id/dragon_value"
+                    style="@style/TextSubHead"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginEnd="@dimen/margin_large"
+                    android:gravity="end"
+                    tools:text="5" />
+            </LinearLayout>
+        </LinearLayout>
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <!-- This section will be filled in by code in the adapter delegate at runtime -->
+    <androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        android:id="@+id/card_view"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_gravity="center"
+        card_view:cardCornerRadius="0dp"
+        android:layout_weight="4"
+        android:clipChildren="false"
+        card_view:cardElevation="20dp"
+        card_view:cardPreventCornerOverlap="false">
+
+        <LinearLayout
+            android:id="@+id/skill_layout"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="#1a1a1a"
```

Comment:
Use a standard color value. This is the first time we're using cards in the app so this will be our standard card background color (and it should probably be darker. I don't remember what color I used in zeplin)

Reply:
You used the color #1a1a1a in Zeplin, but that color isn't used anywhere in our app. I set it to use primaryDarkColor now. If you find that it looks odd, please change it.